Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml

```diff
@@ -0,0 +1,561 @@
+id: Local Analysis alert Investigation - Test
+version: -1
+name: Local Analysis alert Investigation - Test
+description: |-
+  This playbook tests the 'Local Analysis alert Investigation' playbook, which is part of the 'Core' pack. The playbook includes the following conducted tests:.
```

Comment:
Please add a description to the playbook that outlines the specific tests it performs